iHome/ihome.frontend/src/app/services/rooms-api.service.ts

```diff
@@ -3,66 +3,83 @@ import { HttpClient } from '@angular/common/http';
 import { Room } from '../models/room';
 import { User } from '../models/user';
 import { AppSettingsService } from './app-settings.service';
+import { Bill } from '../models/bill';
+import { DeviceToConfigure } from '../models/device-to-configure';
+import { Device } from '../models/device';
 
 @Injectable({
   providedIn: 'root'
 })
 export class RoomsApiService {
   private _apiUrl = '';
 
-  constructor(public http: HttpClient, private _appSettings: AppSettingsService) {
-    this._apiUrl = _appSettings.apiUrl;
+  constructor(private _http: HttpClient, private _appSettings: AppSettingsService) {
+    this._apiUrl = this._appSettings.apiUrl;
    }
 
   public getRooms() {
-    return this.http.get<Array<Room>>(this._apiUrl+'/GetRooms');
+    return this._http.get<Array<Room>>(this._apiUrl+'/GetRooms');
   }
 
   public removeRoom(roomId: number){
-    return this.http.post(this._apiUrl+'/RemoveRoom/'+roomId, {});
+    return this._http.post(this._apiUrl+'/RemoveRoom/'+roomId, {});
   }
 
   public addRoom(roomName: string, roomDescription: string){
-    return this.http.post(this._apiUrl+'/AddRoom', {
+    return this._http.post(this._apiUrl+'/AddRoom', {
       roomName: roomName,
       roomDescription: roomDescription,
       roomImage: ""
     });
   }
 
   public getRoomShares(roomId: number){
-    return this.http.get<Array<User>>(this._apiUrl+'/GetRoomUsers/'+roomId);
+    return this._http.get<Array<User>>(this._apiUrl+'/GetRoomUsers/'+roomId);
   }
 
   public getEmailsTest(emailTests: string){
-    return this.http.get<Array<string>>(this._apiUrl + '/GetEmails/' + emailTests);
+    return this._http.get<Array<string>>(this._apiUrl + '/GetEmails/' + emailTests);
```

Comment:
Test?

Reply:
Kurcze to bym musiał zmienić, tu chodzi o to że podajesz frazę i API ci zwraca takie e-maile co zawierają te frazę